src/crimson/os/seastore/transaction_manager.cc

```diff
@@ -63,6 +64,7 @@ TransactionManager::mount_ertr::future<> TransactionManager::mount()
 {
   LOG_PREFIX(TransactionManager::mount);
   cache->init();
+  segment_cleaner->mount(segment_manager);
```

Comment:
why segment_cleaner->mount two times? one in TransactionManager::mkfs, another one in TransactionManager::mount.

Reply:
mkfs unmounts at the end.